mobile/src/components/uam-configure-group/uam-configure-group.tsx

```diff
@@ -86,8 +86,11 @@ const UAMConfigureGroup: FC = () => {
     navigation.navigate(AppScreenName.UAM);
   };
 
-  const handleCancelEdit = async (): Promise<void> => {
-    dispatch(uamGroupEditActions.cancelEdit);
+  const handleCancel = async (): Promise<void> => {
+    if (group) {
+      dispatch(uamGroupEditActions.cancelEdit);
+    }
+
     navigation.navigate(AppScreenName.UAM);
```

Comment:
```suggestion
    if (group) {
      dispatch(uamGroupEditActions.cancelEdit);
      navigation.navigate(AppScreenName.UAM);
    }
```

Reply:
if I do this, the redirect to uam will not work if there is no group for editing

Reply:
on the group creation screen

Reply:
we need the same dispatch as here to reset the create group screen ?

Reply:
Ok, let's leave it as it is